.github/CONTRIBUTING.md

```diff
@@ -86,234 +103,258 @@ git remote add upstream git@github.com:carbon-design-system/carbon.git
-
-Once all revisions to your pull request are complete, someone from Design
-Systems will squash and merge your commits for you.
+Once the changes are complete and your PR is approved, a maintainer will merge
+your changes.
 
 ## FAQ
 
 ### Who can contribute?
 
-Anyone! We mean it. The one and only requirement is you'll need a
-[public GitHub account](https://github.com/join), as all our assets live on
-GitHub.
+Anyone! The only requirement is a public GitHub account as all our assets live
+on GitHub. Sign up on https://github.com.
 
 - **Development:** If coding is your thing, you can help us by contributing bug
-  fixes or community components. Checkout our
-  [Developer Handbook](../docs/developer-handbook.md) to get your dev
-  environment set up, read up on our best practices and more.
-- **Design:** Design contributions can vary from visual assets, UX interactions,
-  motion design, Sketch kit bug fixes and more.
-- **Content:** Our documentation is just as important as our design and code
-  assets. Whether it's updating our current docs, or adding new
-  [patterns](./patterns), anyone can contribute to our
+  fixes or community components. Check out our
+  [Developer Handbook](../docs/developer-handbook.md) for setup and best
+  practices.
+  <!--
+  TODO: Are non-maintainers allowed to contribute design guidance? That seems to
+  be what this point is implying.
+  -->
+  <!-- TODO: Should 'Sketch kit' be 'Figma kit'? -->
+- **Design:** Contribute visual assets, UX interactions, motion design, Sketch
+  kit fixes, and more.
+  <!--
+  TODO: ./patterns doesn't exist. Can this point be deleted? If not, what should
+  be done here?
+  -->
+- **Content:** Whether updating docs or adding new [patterns](./patterns),
+  anyone can contribute to our contributions to our
   [website content](https://github.com/carbon-design-system/carbon-website).
 - **Research:** Carbon is made up of developers and designers, but
   unfortunately, no dedicated researchers. If you're a researcher and have
   findings that you think could improve Carbon users' experience, you're in the
   right place. This kind of contribution is most effective if coupled with
   design and development forces, which would be presented in a GitHub issue and
   subsequent PR.
+  <!--
+  TODO: Is the part about 'no dedicated researchers' actually true? I attended
+  an internal Carbon event either last year or the year before where the amount
+  of research that went into new and existing features was discussed. Something
+  on the order of thousands of hours was saved by conducting user research and
+  aligning component implementations with user needs. Alternatively, should the
+  following point be used instead?
+  -->
+  <!--
+  - **Research:** If you have findings to improve the Carbon user experience,
+    share them—ideally with supporting design and development details in a GitHub
+    issue and subsequent PR.
+  -->
```

Comment:
Yes and no, it has ebbed and flowed. A year ago we did not have a researcher, our PM wore many hats and that research you saw came from them. We currently have one person with a formal researcher title/background!

I love your new paragraph though, we should totally go with that.